action/envelope.go

```diff
@@ -248,3 +250,51 @@ func (elp *envelope) ChainID() uint32 { return elp.chainID }
 
 // SetChainID sets the chainID value
 func (elp *envelope) SetChainID(chainID uint32) { elp.chainID = chainID }
+
+// ToRLP converts native to RlpTransaction
+func (elp *envelope) ToRLP() (RlpTransaction, error) {
+	var (
+		err  error
+		data []byte
+		ab   AbstractAction
+	)
+	switch act := elp.payload.(type) {
+	case *Transfer:
+		return (*Transfer)(act), nil
+	case *Execution:
+		return (*Execution)(act), nil
+	case *CreateStake:
+		ab = act.AbstractAction
+		data, err = act.EncodeABIBinary()
+	case *DepositToStake:
+		ab = act.AbstractAction
+		data, err = act.EncodeABIBinary()
+	case *ChangeCandidate:
+		ab = act.AbstractAction
+		data, err = act.EncodeABIBinary()
+	case *Unstake:
+		ab = act.AbstractAction
+		data, err = act.EncodeABIBinary()
+	case *WithdrawStake:
+		ab = act.AbstractAction
+		data, err = act.EncodeABIBinary()
+	case *Restake:
+		ab = act.AbstractAction
+		data, err = act.EncodeABIBinary()
+	case *TransferStake:
+		ab = act.AbstractAction
+		data, err = act.EncodeABIBinary()
+	case *CandidateRegister:
+		ab = act.AbstractAction
+		data, err = act.EncodeABIBinary()
+	case *CandidateUpdate:
+		ab = act.AbstractAction
+		data, err = act.EncodeABIBinary()
+	default:
+		return nil, errors.Errorf("invalid action type %T not supported", act)
+	}
+	if err != nil {
+		return nil, err
+	}
+	return wrapStakingActionIntoExecution(ab, address.StakingCreateAddr, data)
```

Comment:
we should file a new PR (in iotex-address repro) to define a new common address, and delete the current 9 special addresses

Reply:
oh, I will fix the special addr

Reply:
https://github.com/iotexproject/iotex-address/pull/19